core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -703,11 +709,10 @@ private[spark] class Executor(
   }
 
   private def setMDCForTask(taskName: String, mdc: Seq[(String, String)]): Unit = {
-    MDC.put("taskName", taskName)
-
```

Comment:
can we do `MDC.clear()` at the beginning of this method? It looks safer if we want to make sure we run the task with only the user-specified mdc properties.

Reply:
ok.